### ISSUE TITLE
4541094d - Tiger Woods PGA Tour 12: The Masters

### ISSUE THREAD

**Issue by nickdelaserda** [xenia-project/game-compatibility#1165]
[Marketplace](https://marketplace.xbox.com/en-US/Product/Tiger-Woods-PGA-TOUR-12-The-Masters/66acd000-77fe-1000-9115-d8024541094d)

Tested on (https://github.com/xenia-project/xenia/commit/84b8b444cf2c2544e959c6fc83dd35fa9cd4b14a)
# Issues:
Can Accept the first few disclaimers on EA Servers and Storage devices, then crashes before reaching the Main Menu. If I let the game go into Demo mode, it succesfully loads the low-quality standby screen showing demo gameplay (see screenshots).

# Log:

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3056740/xenia.zip)

# Screenshot(s):
![crash](https://user-images.githubusercontent.com/1335643/55766030-ab4f0480-5a40-11e9-9f39-30d7a91c6d4e.png)

![1](https://user-images.githubusercontent.com/1335643/55766181-73948c80-5a41-11e9-9d60-989f08234d53.png)

![2](https://user-images.githubusercontent.com/1335643/55766184-755e5000-5a41-11e9-9527-6698184e158c.png)

![3](https://user-images.githubusercontent.com/1335643/55766186-768f7d00-5a41-11e9-8d76-1c838d8b3e5b.png)

![4](https://user-images.githubusercontent.com/1335643/55766188-77c0aa00-5a41-11e9-8858-d7eb6716e888.png)

# Labels:
(recommended labels)
state-intro state-title state-load

**Comment by ZolaKluke** [comment]
Does it crash after the start menu but before the main menu?

**Comment by nickdelaserda** [comment]
> Does it crash after the start menu but before the main menu?

Yes. It crashes during the typical EA sports menus. You press start and it attempts to reach the EA servers, checks storage devices, then crashes at DLC searching. It never reaches the Main Menu

**Comment by ZolaKluke** [comment]
Can you try running the game with this custom version and see if it crashes at the start screen still?
https://ci.appveyor.com/api/buildjobs/n1116li160ypa3aj/artifacts/xenia-d3d12-discord-custom.zip

**Comment by nickdelaserda** [comment]
> Can you try running the game with this custom version and see if it crashes at the start screen still?
> https://ci.appveyor.com/api/buildjobs/n1116li160ypa3aj/artifacts/xenia-d3d12-discord-custom.zip

So this version helped a bit but still crashes. I could get to the main menu and navigate through each section. Since I can't log into an X360 Profile (that I am aware of) I can only play 'Play Now'. Everything renders great and the game runs smooth. But after completing the first hole, xenia says "host crash".

Log file:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3061044/xenia.zip)


![custom 4](https://user-images.githubusercontent.com/1335643/55835901-fde4fb00-5aea-11e9-831b-12751a77f6f9.PNG)

![custom 5](https://user-images.githubusercontent.com/1335643/55835905-00dfeb80-5aeb-11e9-89a8-076ee4120ded.PNG)

![custom 1](https://user-images.githubusercontent.com/1335643/55835911-03dadc00-5aeb-11e9-84ad-0678cf0b9271.PNG)

![custom 2](https://user-images.githubusercontent.com/1335643/55835921-089f9000-5aeb-11e9-947a-0d6c5c8f1ffe.PNG)

![custom 3](https://user-images.githubusercontent.com/1335643/55835923-0a695380-5aeb-11e9-8343-608a4f40e8f4.PNG)

![custom 6](https://user-images.githubusercontent.com/1335643/55835925-0c331700-5aeb-11e9-9761-f79779721fa8.PNG)

**Comment by ZolaKluke** [comment]
So this is a game that also has a profiles issue

Labels:
state-gameplay (?) kernel-unimplemented-feature

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/e7cd2ffffa418002ea5e526a3edba734d5f24990

Booted game. Xenia closes when the game looks for DLC after the title screen.

:bird:This game runs on Canary.

![tig](https://user-images.githubusercontent.com/31154606/103385047-7ad5b680-4abe-11eb-8609-6ba52239c8c0.png)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5755928/xenia.zip)

state-title

**Comment by michaelgmcd** [comment]
Hey, I'm able to run the game on Canary, but I'm unable to navigate to "Quick Play". Not sure why. I can click on "Road to the Masters" and navigate up/down through those options. Any ideas why this might happen?

![Screenshot 2021-04-12 110605](https://user-images.githubusercontent.com/8161781/114416912-2511af80-9b7f-11eb-8765-9f9ea9d409bf.png)

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/3a2c7894d2df96381257f30f4512e3ae27018ea5

Completed one round of golf. 60-100FPS with vsync off and shaders cached. Xenia Guest-crashed at the end of the round. Saves work.
🔊The announcer's audio is too fast.

![twtwe (3)](https://user-images.githubusercontent.com/31154606/123036782-395dfe00-d3b3-11eb-97cb-54c5bea0d544.png)

<details>
  <summary>More Screenshots</summary>

![twtwe (1)](https://user-images.githubusercontent.com/31154606/123036804-3ebb4880-d3b3-11eb-8095-e6b217ffb7e9.png)
![twtwe (2)](https://user-images.githubusercontent.com/31154606/123036809-3ebb4880-d3b3-11eb-99ed-c03de7fd8921.png)
![twtwe (4)](https://user-images.githubusercontent.com/31154606/123036810-3f53df00-d3b3-11eb-8d11-6ca9cb1cb5ce.png)
![twtwe (5)](https://user-images.githubusercontent.com/31154606/123036811-3f53df00-d3b3-11eb-9504-b8315750d3f3.png)
![twtwe (6)](https://user-images.githubusercontent.com/31154606/123036812-40850c00-d3b3-11eb-8e99-051058cf0ca9.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/6698921/xenia.zip)

state-gameplay | apu-garbage

**Comment by binga0485** [comment]
Tried on Canary experimental cbe54d7 - same issues as above.
Game plays fine, just the audio is all over the shop.
We have to be close here?

**Comment by disgusdad** [comment]
Tested on main and canary, graphics look fine, audio after starting play is garbled, almost seems like running to fast.

**Comment by MaouYumisu1** [comment]
Tested on main build and canary, gameplay plays solid but crashes after one hole. Audio is still messed up.

**Comment by brad86m** [comment]
Still crashes after playing a hole. This also applies to TW 13, and 14

**Comment by mvan18** [comment]
Tried the game today, I could get through tigers win, create a character, and start the first amateur tournament but crashed after the first hole

**Comment by brad86m** [comment]
> Tried the game today, I could get through tigers win, create a character, and start the first amateur tournament but crashed after the first hole

It's a real shame. RPCS3 also has the same issue with this TW entry, and others too.

**Comment by Azaiu** [comment]
Crashes after every round specifically when the scorecard is displayed.  
It’s only possible to play masters moments where the scorecard is not displayed which operates accordingly.  
This is a very important game for me, together with a childhood friend we used to always play but he’s since passed away.  
If there are any settings adjustments that can help fix it I’d very much appreciate the help.

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/365
